rts/lra/lra-coordinator-jar/src/test/java/io/narayana/lra/security/jwt/GenerateToken.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020, Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.narayana.lra.security.jwt;
+
+import org.eclipse.microprofile.jwt.Claims;
+
+import java.util.HashMap;
+
+/**
+ * A simple utility class to generate and print a JWT token string to stdout. Can be run with:
+ * mvn exec:java -Dexec.mainClass=io.narayana.lra.security.jwt.GenerateToken -Dexec.classpathScope=test
+ */
+public class GenerateToken {
```

Comment:
Why have we got this utility class under the tests? How do I use it for testing normally?

Reply:
This utility would be used to generate the token which would be passed as a authorization token while calling endpoints during testing.